src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -28,60 +25,52 @@ const defaultProps = {
     },
 };
 
-class OnfidoStep extends React.Component {
-    /**
-     * @returns {boolean|*}
-     */
-    canShowOnfido() {
-        return this.props.walletOnfidoData.hasAcceptedPrivacyPolicy && !this.props.walletOnfidoData.isLoading && !this.props.walletOnfidoData.error && this.props.walletOnfidoData.sdkToken;
-    }
+function OnfidoStep({walletOnfidoData}) {
```

Comment:
@fabioh8010 @gedu Guys, what's our approach related `defaultProps`? I've already seen inline default props and a separate objects. So...

Reply:
I think I'll stick to a having a separate `defaultProps` as per https://github.com/Expensify/App/blob/778f7636c9b59f592193ffad892c0d2904bc84da/contributingGuides/STYLE.md?plain=1#L369